cranelift/filetests/filetests/interpreter/isubbin.clif

```diff
@@ -0,0 +1,49 @@
+test interpret
```

Comment:
Lets wait until [this comment](https://github.com/bytecodealliance/wasmtime/pull/3233#discussion_r695667526) is resolved, to see if we do something similar here.